recipe/bld.bat

```diff
@@ -1,14 +1,35 @@
+:: cmd
+echo "Building %PKG_NAME%."
+
 mkdir build_ && cd build_
+if errorlevel 1 exit /b 1
 
-cmake -G "NMake Makefiles" ^
+:: Generate the build files.
+echo "Generating the build files..."
+cmake .. %CMAKE_ARGS% ^
+    -G"Ninja" ^
     -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
     -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
     -DCMAKE_BUILD_TYPE=Release ^
-    -DBUILD_SHARED_LIBS=ON ^
+    -DBUILD_SHARED_LIBS=OFF ^
```

Comment:
maybe patch CMakelists.txt and enable two library target to build both static and shared ?, see https://stackoverflow.com/questions/2152077/is-it-possible-to-get-cmake-to-build-both-a-static-and-shared-library-at-the-sam

Reply:
Sumit pointed me that the problem is that our latest `suitesparse` on `win` is built as static libraries, not dynamic libraries https://github.com/AnacondaRecipes/suitesparse-feedstock/blob/445b48e7e6268895d0db0c33ced9f3da578ecff9/recipe/meta.yaml#L73-L85.

Reply:
yes, so we have static libs on windows on purpose, there was some reason behind it, if so , why not build both, it would be matter of remove the if statement here https://github.com/DrTimothyAldenDavis/SuiteSparse/blob/42de2d841179f601ebbea83fbf8e7ef80c83039e/AMD/CMakeLists.txt#L99 I suppose

Reply:
as discussed upload it to the test channel and all should be fine, so having that it LGTM